sirepo/raydata/scan_monitor.py

```diff
@@ -592,15 +613,32 @@ def _get_detailed_status(catalog_name, rduid):
 
 
 async def _init_catalog_monitors():
-    def _monitor_catalog(catalog_name):
-        assert catalog_name not in _CATALOG_MONITOR_TASKS
-        return asyncio.create_task(_poll_catalog_for_scans(catalog_name))
-
-    if not cfg.automatic_analysis:
-        return
     for c in cfg.catalog_names:
-        _CATALOG_MONITOR_TASKS[c] = _monitor_catalog(c)
-    await asyncio.gather(*_CATALOG_MONITOR_TASKS.values())
+        _AUTOMATIC_ANALYSIS_HANDLER_TASKS.append(
+            pkasyncio.create_task(_handle_automatic_analysis(c))
+        )
```

Comment:
I'm getting an error ```module 'pykern.pkasyncio' has no attribute 'create_task'``` when running the scan monitor on this branch - is ```pkasyncio.create_task()``` a new method, or should this be ```asyncio.create_task()```?

Reply:
Whoops. It's just a wrapper over asyncio.create_task, but here is the branch with the change. https://github.com/radiasoft/pykern/pull/498